starter-code/src/functions-and-arrays.js

```diff
@@ -11,8 +21,19 @@ var words = [
   'crackpot'
 ];
 
-// Calculating a Sum
+for (var i=0; i<words.length; i++){
```

Comment:
La Logica esta bien, pero el For debería estar dentro de la función, y la variable size debería ser definida antes de comenzar el For.